package.json

```diff
@@ -3,8 +3,9 @@
   "description": "Chrome platform status - chromestatus.com",
   "homepage": "https://www.chromestatus.com",
   "private": true,
+  "type": "module",
```

Comment:
Had to add this because upon upgrading the `del` package.

More about it here:
- https://stackoverflow.com/questions/69862766/getting-error-err-require-esm-while-running-gulp-command
- https://gist.github.com/noraj/007a943dc781dc8dd3198a29205bae04